controllers/MappingsController.php

```diff
@@ -34,7 +34,9 @@ public function mappings() {
 		$locale = !empty($_GET['locale'])
 			? \Zotero\Schema::resolveLocale($_GET['locale'])
 			: 'en-US';
-		
+		unset($this->queryParams['format']);
+		header("Content-Type: application/json");
```

Comment:
Doesn't it [default to atom](https://github.com/zotero/dataserver/blob/2cf66c30a3ef05deb83b11ca47b95b59c347b081/model/API.inc.php#L62) for API v2?
 Without these lines, the call to `/items/new?itemType=book` with header `Zotero-API-Version=2` returns `content-type=application/atom+xml` instead of `application/json`, so [one of the tests ](https://github.com/zotero/dataserver/blob/2cf66c30a3ef05deb83b11ca47b95b59c347b081/tests/remote/tests/API/2/MappingsTest.php#L39) for V2 breaks.

Reply:
Yes, these 2 lines can actually be removed. 
The same 2 lines in the `newItem` function are still needed to get the right application/json content type for API v2